test/run_test.py

```diff
@@ -0,0 +1,253 @@
+"""Test retrace-server before make install.
+
+run: python run_test.py [--coredump="/path/to/the/coredump"
+        --executable="executable" --package="package"]
+        [--delete_repo] [--dont_create_repo]
+        --release="Fedora release 24 (Twenty Four)"
+When coredump path is passed, executbale and package are required
+When no coredump path is passed, coredump will be generated and executable
+   and package are ignored
+Warning: using delete_repo will remove repo, even though it was not created
+        by this script, be careful if you use your own repo
+Note: might need sudo privileges due to using /var/spool/
+"""
+
+import os
+import argparse
+import sys
+import shutil
+sys.path.append("./../src")
+from retrace import *
+
+
+def create_repo(packages, releaseid, version):
+    """Create repo with packages passed in arg packages."""
+    print ("Creating repo")
+
+    conf = config.Config()
+
+    repo_path = os.path.join(conf["RepoDir"], releaseid)
+    if not os.path.isdir(repo_path):
+        os.mkdir(repo_path)
+    needed_packages =\
+                  ["abrt-addon-ccpp", "shadow-utils", "gdb", "rpm"] + packages
+
+    # test if dnf is present, if not, use yum instead
+    c = Popen("dnf --version", shell=True, stdout=PIPE, stderr=PIPE)
+    c.communicate()
+    if c.returncode == 0:
+        cmd_download = 'dnf --releasever=' + version +\
+                       ' --enablerepo=\*debuginfo\* -y' +\
+                       ' --installroot=' + repo_path + ' download --resolve\
+                        --destdir ' + repo_path
+    elif c.returncode == 127:  # 127-> command not found, use yum
+        cmd_download = 'yum --downloadonly --releasever=' +\
+                       version + ' --installroot=' + repo_path +\
+                       ' --enablerepo=\*debuginfo\* -y'+\
+                       ' --downloaddir='+repo_path + ' install'
+    else:
+        print ("Error: problem occurred while looking for dnf or yum")
+        sys.exit(1)
+
+    # add all basic packages
+    for pck in needed_packages:
+        cmd_download = cmd_download + " "+pck
```

Comment:
I believe you can replace this for loop with something more pythonish.

`cmd_download += " %s" % (" ".join(pck))`